src/schema/v2/image/proxies/embedly.ts

```diff
@@ -1,25 +1,32 @@
 import urljoin from "url-join"
 import qs from "qs"
 import config from "config"
+import { Mode } from "."
```

Comment:
I _think_ this kind of import path (`.`) will blow up on linux

Reply:
ah good call